### PR TITLE
Added Windows support to the script and updated the documentation

### PR DESCRIPTION
**Pull request by NickyBoy89**
I was trying help a friend use the bot on a Windows computer, but he was having some issues making changes to the code, so I added the webdrivers, an additional webdriver path tag to specify the driver path, and updated the documentation to evenly distrubute the setup options between operating systems

Feel free to suggest any changes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,69 @@
...
    `Please note that this script was written with the US site in mind, so Nike sites for other countries will likely cause problems.`
    This is because the purchase page changes to where the buy button redirects to a seperate checkout page (as opposed to a direct buy popup during release).
 It is a commandline script written solely in python. `Please run with python 3.7.`
-There are 4 selenium drivers in the bin directory for both Chrome and Firefox on both Linux and MacOS. `I have not added the drivers for Windows.`
+There are 6 selenium drivers in the bin directory for both Chrome and Firefox on Linux, MacOS, and Winows. `The drivers for other operating systems need to be installed from the instructions below.`
 I have found the Firefox driver for MacOS works best.
 
 Ideally, some pieces (or all?) of this could be replaced with direct Nike API requests instead of Selenium. However, I've found that Nike APIs are not very straightforward. 
 
-Before running, make sure to install the requirements by running: `pip install -r requirements.txt`
```
Comment (alexschimpf) [inline README.md:14]: Prolly should keep this line in the README.

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-pause==0.1.2
-python-dateutil==2.7.3
-selenium==3.14.0
-six==1.11.0
-urllib3==1.24.2
+pause>=0.1.2
```
Comment (alexschimpf) [inline requirements.txt:1]: I'm not a big fan of solely using >= for versions. It's only a matter of time before one of these has a new release with a breaking change. If the latest version for all these works, then let's just cap it at whatever that major version is for now (e.g. pause>=0.1.2,<1.0).

File: README.md (hunk 1 above)
Comment (NickyBoy89) [reply README.md:14]: Woop, don't know how that slipped by! Added back in

File: requirements.txt (hunk 2 above)
Comment (NickyBoy89) [reply requirements.txt]: I see your point. I just changed it back to what it was, as this seems like a matter of preference, and you made your original choice for a reason.

**Review by alexschimpf — APPROVED** [review]

**Pull request merged**